mico-core/src/main/java/io/github/ust/mico/core/service/MicoKubernetesClient.java

```diff
@@ -240,10 +240,10 @@ public boolean isApplicationDeployed(MicoApplication micoApplication) throws Kub
      * Checks if a MICO service is already deployed.
      *
      * @param micoService the {@link MicoService}
-     * @return if true the servicegit s is deployed.
+     * @return if true the mico services is deployed.
```

Comment:
Use instead: `if true the {@link MicoService} is deployed.`

Reply:
ok